### PR TITLE
Ansiblize bash that creates DHCP log file

### PR DESCRIPTION
**Pull request by c-mart**
Originally reported by @amit4111989 and @edwins.

### CODE REVIEW

**Comment by c-mart** [comment]
Confirmed this works in a test environment.

**Comment by c-mart** [comment]
Can someone review and merge? @amit4111989, @steve-gregory ?

**Comment by amit4111989** [comment]
i went over the code with my limited Ansible knowledge, and it looks fine to me.

File: ansible/roles/atmo-dhcp/tasks/main.yml [hunk 1]
```diff
@@ -18,14 +18,19 @@
   tags: install-dig
 
 - name: atmo-dhcp - create atmo dhcp log file
-  shell: mkdir -p /var/log/atmo && touch /var/log/atmo/dhcp_hostname.log
+  file:
+    path: /var/log/atmo
+    state: directory
```
Comment (steve-gregory) [inline ansible/roles/atmo-dhcp/tasks/main.yml]: :trophy:

**Pull request merged**